components/email_aliases/BUILD.gn

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//mojo/public/tools/bindings/mojom.gni")
+
+mojom_component("mojom") {
+  output_prefix = "email_aliases_mojom"
+  macro_prefix = "EMAIL_ALIASES_MOJOM"
+
+  generate_java = true
+  generate_legacy_js_bindings = true
```

Comment:
do we need this @petemill ?

Reply:
We definitely need it sometimes - I think sometimes if its shared between upstream WebUI and ours. That said, it looks like

`brave/components/commands/common/BUILD.gn` doesn't use it for its mojom, and its used in the same place. Could you test whether it works without this @arthuredelstein? It might generate a slightly different path for the mojom file, I don't 100% remember.

Reply:
I was able to remove it. The path moved from `mojo_js` to `mojo`.

Reply:
Actually I spoke to soon. When I remove `generate_legacy_js_bindings` it fails.